python/lsst/ts/cbp/component.py

```diff
@@ -172,8 +175,11 @@ async def update_in_position(self):
             elevation=abs(self.elevation - self.target.elevation)
             < self.error_tolerance,
             mask=self.mask == self.target.mask,
-            mask_rotation=abs(self.mask_rotation - self.target.mask_rotation)
-            < self.error_tolerance * 5,
+            mask_rotation=1
+            - math.cos(
+                np.deg2rad(self.mask_rotation) - np.deg2rad(self.target.mask_rotation)
```

Comment:
Can we use math.radians instead of numpy so that we remove dependency on numpy.

Reply:
No problem. But can I ask why we wouldn't want to use numpy?

Reply:
Yeah, numpy is a tricky package to depend on especially with the v1 to v2 transition and I think that since we only want to convert degrees to radians, we can use the math.radians to avoid dependency hell. There is nothing inherently wrong with using numpy, much of our software does use it but it does make packaging more complicated.